### ISSUE TITLE
Operational Excellence Labs not alligned with new UI

### ISSUE THREAD

**Issue by rventuri76** [awslabs/aws-well-architected-labs#283]
For instance step 4.1 at  https://wellarchitectedlabs.com/operational-excellence/100_labs/100_inventory_patch_management/4_inventory_mgmt/ refers to "Managed Instances" while probably now is Manage Fleet. There are many other such things that makes it hard to follows the steps.

**Comment by TheRealDeomax** [comment]
Thank you for pointing that out! I was trying to find it for the life of me!

**Comment by J-Arinam** [comment]
I agree its harder to follow up to the system manager point. Please update for the new UIs throughout the labs

**Comment by rswift** [comment]
The [Fleet Manager UI](https://console.aws.amazon.com/systems-manager/managed-instances) (note the URL ends `/managed-instances`) shows a banner message at the top that reads:
> Managed Instances moved to a new Systems Manager capability called Fleet Manager. You can find all features of managed instances in Fleet Manager.
